client/state/reader/site-dismissals/reducer.js

```diff
@@ -1,25 +0,0 @@
-import { READER_DISMISS_SITE, READER_DISMISS_POST } from 'calypso/state/reader/action-types';
-import { combineReducers } from 'calypso/state/utils';
-
-export const items = ( state = {}, action ) => {
-	switch ( action.type ) {
-		case READER_DISMISS_SITE: {
-			return {
-				...state,
-				[ action.payload.siteId ]: true,
-			};
-		}
-		case READER_DISMISS_POST: {
-			return {
-				...state,
-				[ action.payload.siteId ]: true,
-			};
-		}
-	}
-
-	return state;
-};
```

Comment:
The `dismissSite` and `dismissPost` actions are still used, aren't they? But they are fire-and-forget actions that only need to send the REST requests, and they don't need to store any Redux state anymore, is that right?

I'd consider removing all the data-layer abstractions and converting them to simple thunks:
```js
function dismissSite( siteId ) {
  return ( dispatch ) => wpcom.req.post( '/me/dismiss/...' ).then( () => dispatch( successNotice() ) );
}
```

Reply:
Sounds good, but this feels more like work for another PR. I'd be happy to follow up.